source/platforms/github/GitHubGit.ts

```diff
@@ -114,7 +113,7 @@ export default async function gitDSLForGitHub(api: GitHubAPI): Promise<GitDSL> {
       // added or removed. This makes it really easy to act on specific
       // changes to JSON DSLs
 
-      if (isarray(diff.after) && isarray(diff.before)) {
+      if (Array.isArray(diff.after) && Array.isArray(diff.before)) {
```

Comment:
I think we should be safe using the native `Array.isArray()` instead of that deprecated lodash package. 👍